docs/admin/high-availability.md

```diff
@@ -6,19 +6,27 @@ endpoint. [GCP](https://cloud.google.com/sql/docs/postgres/high-availability), [
 and other cloud vendors offer fully-managed HA Postgres services that pair
 nicely with Coder.
 
-For Coder to operate correctly, every node must be within 10ms of each other
-and Postgres. We make a best-effort attempt to warn the user when inter-Coder
-latency is too high, but if requests start dropping, this is one metric to investigate.
+For Coder to operate correctly, Coderd instances should have low-latency connections
+to each other so that they can effectively relay traffic between users and workspaces no
+matter which Coderd instance users or workspaces connect to. We make a best-effort attempt
+to warn the user when inter-Coderd latency is too high, but if requests start dropping, this
```

Comment:
What do we consider "too high" for the health check? We might as well document this threshold as well?

Reply:
TBH, I'm not sure what this is referring to.  @kylecarbs do you know?

Reply:
@bpmct this text is not being changed by this PR, just whitespace.  I understand the desire to get better documentation around the health check, but it's scope creep for this PR.  Can we agree on whether or not the _changes_ this PR is making are appropriate so it can get merged?

Reply:
Ah I misunderstood it as a text change.